src/main/java/org/jenkinsci/plugins/gitclient/JGitAPIImpl.java

```diff
@@ -1112,6 +1112,14 @@ public ChangelogCommand includes(ObjectId rev) {
                 }
             }
 
+            public ChangelogCommand firstParent() {
+                return firstParent(true);
+            }
+
+            public ChangelogCommand firstParent(boolean firstParent) {
+                return this;
```

Comment:
Please either implement this functionality in JGit (preferred) or log a warning that this parameter is not supported in the JGit implementation. Otherwise the user has no idea when enabling this feature why it's not working as expected.

Reply:
thanks for the suggestion, I addressed this in my latest commit